docs/datastores.md

```diff
@@ -0,0 +1,59 @@
+Datastores
+==========
+
+Version 1.0+ supports using Postgres as a storage backend instead of the
+legacy NeDB storage backend. If you wish to take advantage of this, please
```

Comment:
Take out the `If you wish...` sentence: we immediately go into discussions about EOL, so it's important that people feel like they need to upgrade.

Also:
```suggestion
deprecated NeDB storage backend. If you wish to take advantage of this, please
```